plio/signals.py

```diff
@@ -28,3 +28,11 @@ def item_update_cache(sender, instance, **kwargs):
 def question_update_cache(sender, instance, **kwargs):
     # invalidate saved cache for the plio
     invalidate_cache_for_instance(instance.item.plio)
+
+
+@receiver([post_save], sender=Question)
+def question_delete_image_on_deletion(sender, instance, **kwargs):
```

Comment:
Can you also add one liner of docstring? Because in the code, that part is confusing
`if instance.deleted is not None` -- I mean we're doing this when the question instance is deleted right? and the code says `if instance.deleted is not None`

Reply:
soft_delete mention